homework/max-of-vector/maxOfVector.hpp

```diff
@@ -1,8 +1,10 @@
 #pragma once
+
+#include <algorithm>
 #include <vector>
-#include <limits>
 
 int maxOfVector(const std::vector<int>& vec) {
-    // TODO: Implement me :)
-    return {};
+    auto greatest{ std::max_element(vec.begin(), vec.end()) };
+
+    return greatest != vec.end() ? *greatest : 0;
```

Comment:
More readable will be:

```C++
const auto it = std::max_element(vec.begin(), vec.end());
return it != vec.end() ? *it : 0;
```

What's more this will be faster, because you will call function `max_element` once. In Your example you sometimes call it twice.